wagtail/admin/templates/wagtailadmin/shared/editing_sessions/list.html

```diff
@@ -0,0 +1,124 @@
+                        </div>
+
+                        {% fragment as refresh_button_content %}
+                            {% icon name="rotate" %}
+                            {% trans "Refresh" %}
+                        {% endfragment %}
+                        {% dialog_toggle classname="button button-small button-secondary chooser__choose-button" text=refresh_button_content dialog_id="w-unsaved-changes-dialog" %}
+                    </div>
+                {% enddropdown %}
+            {% elif session.is_editing %}
+                {% if request.user == session.user %}
+                    {% blocktranslate trimmed asvar has_unsaved_changes_message %}
+                        You have unsaved changes in another session
+                    {% endblocktranslate %}
+                {% else %}
+                    {% blocktranslate trimmed with user_name=session.user|user_display_name|default:_("system") asvar has_unsaved_changes_message %}
+                        {{ user_name }} has unsaved changes
+                    {% endblocktranslate %}
+                {% endif %}
+
+                {% dropdown theme="popup" classname="w-editing-sessions__session w-editing-sessions__session--editing" toggle_label=avatar toggle_aria_label=has_unsaved_changes_message|capfirst %}
+                    <div class="w-editing-sessions__popup">
+                        <div class="w-editing-sessions__message">
+                            {% icon name="warning" %}
+                            {{ has_unsaved_changes_message|capfirst }}
+                        </div>
+                    </div>
+                {% enddropdown %}
+            {% else %}
+                <button class="w-editing-sessions__session" type="button" data-controller="w-tooltip">
+                    {{ avatar }}
+                    <div class="w-editing-sessions__popup" data-w-tooltip-target="content" hidden>
+                        <span class="w-editing-sessions__name">{{ session.user|user_display_name|default:_("system")|capfirst }}</span>
+                        <span class="w-sr-only">-</span>
+                        <span>{% trans "Currently viewing" %}</span>
+                    </div>
+                </button>
+            {% endif %}
+        </li>
+    {% endfor %}
+    {% if sessions|length > 4 %}
+        {% blocktranslate trimmed count num_user=sessions|length|add:"-4" asvar num_other_users_message %}
+            One other user is currently viewing
+        {% plural %}
+            {{ num_user }} other users are currently viewing
+        {% endblocktranslate %}
+
+        {% fragment as more_sessions_toggle %}
+            +{{ sessions|length|add:"-4" }}
+        {% endfragment %}
+
+        {% dropdown theme="drilldown" classname="w-editing-sessions__session w-editing-sessions__session--more" toggle_classname="w-editing-sessions__avatar" toggle_label=more_sessions_toggle toggle_aria_label=num_other_users_message|capfirst %}
```

Comment:
I’ve not tested this yet but want to flag we probably shouldn’t use both `toggle_label` and `toggle_aria_label`. It’d be a problem for voice recognition users trying to name the button.

Reply:
I've added a commit that uses `toggle_describedby` instead.